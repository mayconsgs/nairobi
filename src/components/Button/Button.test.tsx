import { Button } from ".";
import { render } from "../../utils/test";

describe("<Button />", () => {
  it("render Button", () => {
    const { container } = render(<Button>Button</Button>);
    expect(container).toMatchSnapshot();
  });

  it("render Raised Button", () => {
    const { container } = render(<Button variant="raised">Button</Button>);
    expect(container).toMatchSnapshot();
  });

  it("render Outline Button", () => {
    const { container } = render(<Button variant="outline">Button</Button>);
    expect(container).toMatchSnapshot();
  });
});
