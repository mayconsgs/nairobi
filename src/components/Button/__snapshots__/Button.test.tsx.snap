// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> render Button 1`] = `
<div>
  <button
    class="sc-bdvvtL eqrBTb"
  >
    Button
  </button>
</div>
`;

exports[`<Button /> render Outline Button 1`] = `
<div>
  <button
    class="sc-bdvvtL eGTWdX"
  >
    Button
  </button>
</div>
`;

exports[`<Button /> render Raised Button 1`] = `
<div>
  <button
    class="sc-bdvvtL eqrBTb"
  >
    Button
  </button>
</div>
`;
